src/main/java/com/cinema/model/MovieSession.java

```diff
@@ -0,0 +1,87 @@
+package com.cinema.model;
+
+import java.time.LocalDateTime;
+import java.util.Objects;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "movie_session")
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne
+    private Movie movie;
+    @OneToOne
+    @JoinColumn(name = "cinema_hall_id")
+    private CinemaHall cinemaHall;
+    private LocalDateTime showTime;
```

Comment:
```suggestion
    private LocalDateTime showTime;
```
add please here @ JoinColumn(name "show_time")